templates-generator/vue-config.js

```diff
@@ -0,0 +1,136 @@
+module.exports = {
+    sourcePath: 'packages/devextreme-cli/testing/sandbox/vue/my-app/',
+    targetPath: 'packages/devextreme-cli/templates/vue/',
+    sourceGlob: '**/*.{js,scss,json,vue}',
+    ignoreList: [
+        '{node_modules,public,src/themes/generated}/**/*.*',
+        'src/{App.test,setupTests,serviceWorker,index}.js',
+        'src/components/HelloWorld.vue',
+        '{package,package-lock}.json',
+        'babel.config.js'
+    ],
+    replaceRules: [
+        {
+            glob: 'src/themes/metadata.additional.json',
+            definitions: [
+                {
+                    before: /"baseTheme": "[^"]*"/,
+                    after: '"baseTheme": "material.orange.dark"'
+                }
+            ]
+        },
+        {
+            glob: 'src/themes/metadata.base.json',
+            definitions: [
+                {
+                    before: /"baseTheme": "[^"]*"/,
+                    after: '"baseTheme": "material.orange.light"'
+                }
+            ]
+        },
+        {
+            glob: 'src/themes/metadata.*.json',
+            definitions: [
+                {
+                    before: /"items": \[[^\]]*]/,
+                    after: '"items": []'
+                }
+            ]
+        },
+        {
+            glob: 'src/app-info.js',
+            definitions: [
+                {
+                    before: 'My App',
+                    after: '<%=project%>'
+                }
+            ],
+        },
+        {
+            glob: 'src/app-navigation.js',
+            definitions: [
+                {
+                    before: /\[(.*?)\];/s,
+                    after: '[<%=^empty%>$1<%=/empty%>];'
+                }
+            ]
+        },
+        {
+            glob: 'src/router.js',
+            definitions: [
+                {
+                    before: /(import Home .*?)(import defaultLayout[^;]*)/s,
+                    after: '<%=^empty%>$1<%=/empty%>$2'
+                },
+                {
+                    before: /side-nav-(inner|outer)-toolbar/,
+                    after: '<%=layout%>'
+                },
+                {
+                    before: /(,)(\s+{\s+path: "\/".*redirect: "\/home"[^}]*})/s,
+                    after: '$1<%=^empty%>$2<%=/empty%>'
+                },
+                {
```

Comment:
Why did you move this block to the top? It looks like it breaks the sequence logic.

Reply:
Also, it could leave a redundant comma at the end.

Reply:
I replaced and changed this expression